src/shared/sources/_lib/validate-source.js

```diff
@@ -88,8 +88,19 @@ function validateSource (source) {
         ))
       }
 
+      requirement(url || paginated, datedError('Require url or paginated'))
```

Comment:
Error is not especially clear

Reply:
Changed to `requirement(url || paginated, datedError('Crawler must have either url or paginated'))`